src/synthetic_population_generator/cli.py

```
@@ -590,6 +757,179 @@ def combine_datasets(ctx, input_file, output_file):
+             report_html, output_json_metrics):
+    """
+    Evaluates the quality of synthetic data against real data using SDV metrics.
+
+    Compares a synthetic dataset to a real dataset, optionally using provided SDV
+    metadata. It can generate an HTML report with detailed metrics and visualizations,
+    save raw metrics to a JSON file, and perform cohort-specific evaluations if
+    cohort definitions are provided. If no output file paths are specified, a summary
+    of key overall scores is printed to the console.
+    """
+    logger.info(f"Executing 'evaluate'. Real data: {real_data}, Synthetic data: {synthetic_data}")
+    if metadata: logger.info(f"Using metadata file: {metadata}")
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)